### ISSUE TITLE
whats faster coral TPU vs khadas NPU 

### ISSUE THREAD

**Issue by taloot** [OAID/Tengine#798]
i have hard time googling and looking for information regarding this,
i used x86 with coral tpu, and i got 40ms on yolov5s512x512
and i have khadas in hand,, but i feel that khadas even if its 5Tops, and coral is 4Tops. but in real coral tpu faster.
this is my test on yolov5s...512x512 i want to see equivalent using  the am logic NPU or any another npu 
https://www.youtube.com/watch?v=D9IExho8pwo
i hope someone show me better results than this

**Comment by BUG1989** [comment]
Hi, @taloot thanks for your issue.

- what is the inference time of yolov5s at 512x512 with Khadas VIM3 NPU?
- some type op in yolov5s is not friendly for Khadas VIM3's NPI, we have a optimize network of yolov5s 640x640, it's cost 50ms with Khadas VIM3 NPU;
- If you have needs, we can help you design a more optimized network using Khadas VIM3 NPU.

**Comment by taloot** [comment]
> Hi, @taloot thanks for your issue.
> 
> * what is the inference time of yolov5s at 512x512 with Khadas VIM3 NPU?
> * some type op in yolov5s is not friendly for Khadas VIM3's NPI, we have a optimize network of yolov5s 640x640, it's cost 50ms with Khadas VIM3 NPU;
> * If you have needs, we can help you design a more optimized network using Khadas VIM3 NPU.

 yes please i need to port the model i have for TPU, and compare it with coral tpu

**Comment by taloot** [comment]
> Hi, @taloot thanks for your issue.
> 
> * what is the inference time of yolov5s at 512x512 with Khadas VIM3 NPU?
> * some type op in yolov5s is not friendly for Khadas VIM3's NPI, we have a optimize network of yolov5s 640x640, it's cost 50ms with Khadas VIM3 NPU;
> * If you have needs, we can help you design a more optimized network using Khadas VIM3 NPU.

can u show me a working video for this

**Comment by AkkiSony** [comment]
@taloot Hi,
I see that you are working with VIM3 NPU and Coral TPU. Did you get to know which one is faster and better in terms of performance. I am working with the same AI accelerators.

Also, did you try working with other architectures other than yolov'x' ?
Thanks in advance! :)

**Comment by taloot** [comment]
the vim3 didnt work for me at all
coral tpu works and every day it become better 
me i thought the npu performance will be better, but i found out that its impossible to get the yolov5 running on this npu..
also as an advice for you.. go for x86 soc (j5005) and coral tpu/s and u can add as many as u want
also cheap. 
 if vim3 works as they clam,, u should see real test for it on the modern models.. it works only on obsoelet  or useless models 
u cant relay on it in production system, also u can see the @BUG1989 run away when i asked him for help... if it works he should reply. but as i said it  only works on obsolete models 
also im highly recommend  for you to stay away from khadas, for many resons,
1, support is poor
2. expansive
3. hard to get the kernel u need nda
